free-2.md

```diff
@@ -5,47 +5,105 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+    This user would basically be anyone that cooks, and the language would be
+    a good way to keep track of any recipes and any alterations they may need
+    to make to it. Kind of like a cook book!
+    They would probably need to also have some familiarity with programming.
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+    The need met is to have a way to store cookbooks and recipes as a DSL/text
+    file. I mean Chef language already exists, but I like the idea of having
+    a collection of recipes stored in a cookbook as functions and variables. 
+    The person probably just looks up on their phone or just cooks as is. If
+    I could help them, they can define different dishes/processes to make dishes
+    and look them up that way.
```

Comment:
The best part of this is being able to search and apply modifications to the structure that holds all the recipes, searching in a much more reasonable way.